README.rst

```diff
@@ -15,12 +15,61 @@ to generate a client library.
 Purpose
 -------
 
-This library primarily exists to facilitate experimentation, particularly
-regarding:
+This library aims to replace the `monolithic generator`_
+with some improvements:
 
 - An explicit normalized format for specifying APIs.
 - Light weight, in-language code generators.
 
+.. _monolithic generator: https://github.com/googleapis/gapic-generator
+
+
+Bazel
+-------------
+This generator can be called from Bazel, which is a recommended way of using it inside a continuous integration build or any other automated pipeline.
+
+Clone the repository
+$ git clone https://github.com/googleapis/googleapis.git
+
+Create the targets
+------------------
+You need to add the following targets to your BUILD.bazel file.
+
+load(
+    "@gapic_generator_python//rules_python_gapic:py_gapic.bzl",
+    "py_gapic_library"
+)
+
+load(
+    "@gapic_generator_python//rules_python_gapic:py_gapic_pkg.bzl",
+    "py_gapic_assembly_pkg"
+)
+
+py_gapic_library(
+    name = "documentai_py_gapic",
+    srcs = [":documentai_proto"],
+)
+
+py_gapic_assembly_pkg(
+    name = "documentai-v1beta2-py",
+    deps = [
+        ":documentai_py_gapic",
+    ],
+)
```

Comment:
Perhaps we could also show examples of the `bazel` build command, as well as examples using `protoc`